rpcs3/Emu/RSX/GL/GLPresent.cpp

```diff
@@ -141,6 +141,11 @@ void GLGSRender::flip(const rsx::display_flip_info_t& info)
 		buffer_width = std::min(buffer_width, avconfig.resolution_x);
 		buffer_height = std::min(buffer_height, video_frame_height);
 	}
+	else if (!buffer_width)
```

Comment:
std::min is used above, if it can be 0 it will also yield zero.

Reply:
It cannot be zero.

Reply:
cellVideoOutConfigure is not calling sys_rsx_context_attribute, display_buffers data may not be initailized even if OutConfigure was called,

Reply:
i.e. `buffer_height = std::min(buffer_height, video_frame_height)` will result in 0 if it sys_rsx syscalll was not called.